traffic_portal/app/src/common/modules/form/deliveryService/edit/FormEditDeliveryServiceController.js

```diff
@@ -202,10 +202,13 @@ var FormEditDeliveryServiceController = function(deliveryService, origin, type,
 				if (options.status.id == $scope.COMPLETE) {
 					createDeliveryServiceUpdateRequest(dsRequest, options.comment, true).then(
 						function() {
-							deliveryServiceService.updateDeliveryService(deliveryService);
-						}).then(function() {
-							$state.reload(); // reloads all the resolves for the view
-							messageModel.setMessages([ { level: 'success', text: 'Delivery Service [ ' + deliveryService.xmlId + ' ] updated' } ], false);
+							deliveryServiceService.updateDeliveryService(deliveryService).
```

Comment:
I mistakenly branched off another branch that has a PR that has yet to be merged. Once PR#4273 is merged. This should go away.